ee/middleware/upstreamoauth/model.go

```diff
@@ -0,0 +1,48 @@
+package upstreamoauth
+
+import (
+	"time"
+
+	"github.com/TykTechnologies/tyk/apidef"
+	"github.com/TykTechnologies/tyk/ctx"
+	"github.com/TykTechnologies/tyk/internal/httpctx"
+	"github.com/TykTechnologies/tyk/internal/model"
+)
+
+const (
+	ErrorEventName                 = "UpstreamOAuthError"
+	MiddlewareName                 = "UpstreamOAuth"
+	ClientCredentialsAuthorizeType = "clientCredentials"
+	PasswordAuthorizeType          = "password"
```

Comment:
```suggestion
	PasswordAuthorizeType          = "Password"
```

Casing doesn't match the others

Reply:
it was decided to be lowercase password in the contract